File: Plugins/Aspose-Cells-Java-for-Jython/asposecells/WorkingWithWorksheets/DisplayHideGridlines.py
from asposecells import Settings
from com.aspose.cells import Workbook


class DisplayHideGridlines:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithWorksheets/DisplayHideGridlines/'
     
        workbook = Workbook(dataDir + "Book1.xls")
        
        #Accessing the first worksheet in the Excel file
        worksheets = workbook.getWorksheets()

        worksheet = worksheets.get(0)

        #Hiding the grid lines of the first worksheet of the Excel file
        worksheet.setGridlinesVisible(False)

        #Saving the modified Excel file in default (that is Excel 2000) format
        workbook.save(dataDir + "output.xls")

        # Print message
        print "Grid lines are now hidden on sheet 1, please check the output document."
 
if __name__ == '__main__':        
    DisplayHideGridlines()

File: Examples/src/main/java/com/aspose/cells/examples/worksheets/DisplayHideGridlines.java
package com.aspose.cells.examples.worksheets;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class DisplayHideGridlines {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(DisplayHideGridlines.class) + "worksheets/";

		// Instantiating a Workbook object by excel file path
		Workbook workbook = new Workbook(dataDir + "book1.xls");

		// Accessing the first worksheet in the Excel file
		WorksheetCollection worksheets = workbook.getWorksheets();

		Worksheet worksheet = worksheets.get(0);

		// Hiding the grid lines of the first worksheet of the Excel file
		worksheet.setGridlinesVisible(false);

		// Saving the modified Excel file in default (that is Excel 2000) format
		workbook.save(dataDir + "DisplayHideGridlines_out.xls");

		// Print message
		System.out.println("Grid lines are now hidden on sheet 1, please check the output document.");

	}
}
